tests/individual/TimeTravelFrozen.cpp

```diff
@@ -0,0 +1,118 @@
+#define CATCH_CONFIG_MAIN
+#include <catch.hpp>
+#include <future>
+#include <nuclear>
+
+#include "test_util/TestBase.hpp"
+
+namespace {
+
+constexpr std::chrono::milliseconds EVENT_1_TIME = std::chrono::milliseconds(4);
+constexpr std::chrono::milliseconds EVENT_2_TIME = std::chrono::milliseconds(8);
+
+struct WaitForShutdown {};
+
+class TestReactor : public test_util::TestBase<TestReactor, 5000> {
+public:
+    // Start time of steady clock
+    std::chrono::steady_clock::time_point steady_start_time = std::chrono::steady_clock::now();
+
+    // Time travel action
+    NUClear::message::TimeTravel::Action action = NUClear::message::TimeTravel::Action::RELATIVE;
+
+    // Time adjustment
+    NUClear::clock::duration adjustment = std::chrono::milliseconds(0);
+
+    // Real-time factor
+    double rtf = 1.0;
+
+    // Events
+    std::vector<std::string> events = {};
+
+    TestReactor(std::unique_ptr<NUClear::Environment> environment) : TestBase(std::move(environment), false) {
+
+        on<Startup>().then([this] {
+            // Emit a chrono task to run at time EVENT_1_TIME
+            emit<Scope::DIRECT>(std::make_unique<NUClear::dsl::operation::ChronoTask>(
+                [this](NUClear::clock::time_point&) {
+                    events.push_back("Event 1");
+                    return false;
+                },
+                NUClear::clock::now() + EVENT_1_TIME,
```

Comment:
If we're basing the clocks from zero then give these events an absolute time too
```suggestion
                NUClear::clock::time_point(EVENT_1_TIME),
```